api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -51,17 +51,6 @@ cron:
   schedule: every 30 minutes
   timezone: UTC
   target: api
-- description: >
-    Find billing projects associated with deleted workspaces and transfer ownership to
-    designated "Garbage Collection" Service Accounts. This legacy process works around the lack of
-    API support for deleting these billing projects directly.
-
-    The Terra delete Account API is now available, and this cron job is deprecated. To be removed
-    as part of RW-3627.
-  url: /v1/cron/billingProjectGarbageCollection
```

Comment:
also rm from cron_perf.yaml

Reply:
Good catch, thanks. Could have sworn I'd gotten that one..